zeebe/gateway-protocol/src/main/proto/v2/cluster-variables.yaml

```diff
@@ -0,0 +1,368 @@
+        "400":
+          $ref: 'common-responses.yaml#/components/responses/InvalidData'
+        "401":
+          $ref: 'common-responses.yaml#/components/responses/Unauthorized'
+        "403":
+          $ref: 'common-responses.yaml#/components/responses/Forbidden'
+        "404":
+          description: Cluster variable not found
+          content:
+            application/problem+json:
+              schema:
+                $ref: 'problem-detail.yaml#/components/schemas/ProblemDetail'
+        "500":
+          $ref: 'common-responses.yaml#/components/responses/InternalServerError'
+    delete:
+      operationId: deleteTenantClusterVariable
+      x-eventually-consistent: false
+      tags:
+        - Cluster Variable
+      summary: Delete a tenant-scoped cluster variable
+      parameters:
+        - name: tenantId
+          in: path
+          required: true
+          schema:
+            $ref: 'identifiers.yaml#/components/schemas/TenantId'
+          description: The tenant ID
+        - name: name
+          in: path
+          required: true
+          schema:
+            type: string
+          description: The name of the cluster variable
+      responses:
+        '204':
+          description: Cluster variable deleted successfully
+        "400":
+          $ref: 'common-responses.yaml#/components/responses/InvalidData'
+        "401":
+          $ref: 'common-responses.yaml#/components/responses/Unauthorized'
+        "403":
+          $ref: 'common-responses.yaml#/components/responses/Forbidden'
+        "404":
+          description: Cluster variable not found
+          content:
+            application/problem+json:
+              schema:
+                $ref: 'problem-detail.yaml#/components/schemas/ProblemDetail'
+        "500":
+          $ref: 'common-responses.yaml#/components/responses/InternalServerError'
+
+components:
+  schemas:
+    ClusterVariableScopeEnum:
+      type: string
+      enum: [ GLOBAL, TENANT ]
+      description: The scope of a cluster variable.
+    CreateClusterVariableRequest:
+      type: object
+      required:
+        - name
+        - value
+      properties:
+        name:
+          type: string
+          description: The name of the cluster variable. Must be unique within its scope (global or tenant-specific).
+        value:
+          type: object
+          description: The value of the cluster variable. Can be any JSON object or primitive value. Will be serialized as a JSON string in responses.
+    ClusterVariableResult:
+      type: object
+      required:
+        - name
+        - value
+        - scope
+      properties:
+        name:
+          type: string
+          description: The name of the cluster variable. Unique within its scope (global or tenant-specific).
+        value:
+          type: string
+          description: The value of the cluster variable as a JSON-serialized string. Regardless of the input type during creation, values are returned as strings.
```

Comment:
❓ Can this value be truncated? If so, should we reflect this in the API? For process variables, we're currently [adding a feature](https://github.com/camunda/camunda/pull/40884) to support fetching the full values when searching for variables if needed. We might have to consider something like that here as well.